concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -127,76 +226,92 @@ writeSection ::
     BlockHash ->
     -- |Height of first block in section
     BlockHeight ->
-    -- |Handle to write to
+    -- |Height of last block in section
+    BlockHeight ->
+    -- |Export directory
+    FilePath ->
+    -- |Index file handle
     Handle ->
-    -- |Action to write out the blocks and return the number of them
-    m Word64 ->
-    -- |Action to write out the finalization records and return the number of them
-    m Word64 ->
+    -- |Chunk size in blocks
+    Word64 ->
     m ()
-writeSection
+writeChunks
     sectionGenesisIndex
     sectionProtocolVersion
     sectionGenesisHash
     sectionFirstBlockHeight
-    hdl
-    writeBlocks
-    writeFinRecs = do
+    sectionLastBlockHeight
+    outDir indexHdl chunkSize = do
+        let chunkName = outDir </> "blocks-"
+                        ++ show sectionGenesisIndex ++ "-"
+                        ++ show sectionFirstBlockHeight
+                        ++ ".dat"
+        chunkHdl <- liftIO $ initialHandle chunkName
```

Comment:
I mentioned this before, but I'm concerned that replacing existing chunks with new ones could cause problems for web caching/CDN. We could avoid this by picking a new name if a chunk already exists.